### DISCUSSION TITLE
Adding a feature in a bucket puts it at the end of the list

### DISCUSSION THREAD

**Discussion by tchapi** [area17/twill#1011] [Ideas]
Hi all,

When you add an item to a bucket, the item you choose in the 'available items' lands at the _end_ of the list on the right.

This is not a problem when you have maybe 5 featured items, but if you have more (_think 30 for instance_), then the item is at the end of the list, and if you want to put it at the front of the list, it's a bit cumbersome : you have to scroll to the bottom, drag it, scroll back top, drop it...

Could there be a setting to put it automatically in the front ?
If you point me to where such a change could be done, I can prepare a PR ;)

Thanks a lot
BR

**Comment by yanhao-li** [comment]
Sounds like a UX enhancement, Twill current doesn't has such setting option to put a bucket item on top.
I would suggest to take a look at the corresponding vue components: `Bucket.vue` `BucketItem.vue`, and see if that's possible to make it true from there.
